beets/library.py

```diff
@@ -130,17 +130,17 @@ def format(self, value):
         return time.strftime(beets.config['time_format'].as_str(),
                              time.localtime(value or 0))
 
-    def parse(self, string):
+    def parse(self, s):
```

Comment:
Let's change these argument names back (because the change is not relevant to the feature at hand).